portfolio/src/main/webapp/backpacking.html

```diff
@@ -6,19 +6,138 @@
+      <a class="next" onclick="advanceSlidesBy(1)">&#10095;</a>
+
+      <!-- Image text -->
+      <div class="caption-container">
+        <p id="caption"></p>
+      </div>
+
+      <div class="row">
+        <div class="column">
+          <img class="demo cursor" src="images/backpacking/bp-1.JPG" 
+          style="width:100%" onclick="setCurrentSlide(1)" 
+          alt="Lake Ediza, Ritter and Banner Peaks; John Muir Trail">
+        </div>
+        <div class="column">
+          <img class="demo cursor" src="images/backpacking/bp-2.JPG" 
+          style="width:100%" onclick="setCurrentSlide(2)" 
+          alt="Ritter and Banner Twin Peaks; John Muir Trail">
+        </div>
+        <div class="column">
+          <img class="demo cursor" src="images/backpacking/bp-4.JPG" 
+          style="width:100%" onclick="setCurrentSlide(3)" 
+          alt="Muir Pass; John Muir Trail">
+        </div>
+        <div class="column">
+          <img class="demo cursor" src="images/backpacking/bp-5.JPG" 
+          style="width:100%" onclick="setCurrentSlide(4)" 
+          alt="Ascent to Dragon's Tooth; Virginia Triple Crown">
+        </div>
+        <div class="column">
+          <img class="demo cursor" src="images/backpacking/bp-6.JPG" 
+          style="width:100%" onclick="setCurrentSlide(5)" 
+          alt="Paradise, Rainier; Mt. Rainier National Park">
+        </div>
+        <div class="column">
+          <img class="demo cursor" src="images/backpacking/bp-9.jpg" 
+          style="width:100%" onclick="setCurrentSlide(6)" 
+          alt="Hoh Rainforest; Olympic Peninsula">
+        </div>
+        <div class="column">
+          <img class="demo cursor" src="images/backpacking/bp-8.jpg" 
+          style="width:100%" onclick="setCurrentSlide(7)" 
+          alt="Hoh Rainforest; Olympic Peninsula">
+        </div>
+        <div class="column">
+          <img class="demo cursor" src="images/backpacking/bp-10.jpg" 
+          style="width:100%" onclick="setCurrentSlide(8)" 
+          alt="Hole in the Wall, Rialto Beach; Shores of the Olympic">
+        </div>
+        <div class="column">
+          <img class="demo cursor" src="images/backpacking/bp-7.jpg" 
+            style="width:100%" onclick="setCurrentSlide(9)" 
+            alt="Hole in the Wall, Rialto Beach; Shores of the Olympic">
+        </div>
       </div>
     </div>
+
+    <script>
+      let slideIndex = 1;
+      showSlides(slideIndex);
+
+      function advanceSlidesBy(incrementBy) {
+        showSlides(slideIndex += incrementBy);
+      }
+
+      function setCurrentSlide(slideNumber) {
+        showSlides(slideIndex = slideNumber);
+      }
+
+      function showSlides(numSlidesToShow) {
+        const slides = document.getElementsByClassName("mySlides");
+        let dots = document.getElementsByClassName("demo");
+        let captionText = document.getElementById("caption");
+        if (numSlidesToShow > slides.length) {slideIndex = 1}
```

Comment:
nit: I would move this to its own line although that's option. What's not optional is it needs a semicolon :)